ui/src/app/kvm/views/KVMDetails/KVMResources/KVMResources.test.tsx

```diff
@@ -1,30 +1,57 @@
-import * as React from "react";
-
 import { mount } from "enzyme";
```

Comment:
This file is unchanged from the original `LxdResources.test.tsx` apart from two new tests regarding the overall resources card.